README.md

```diff
@@ -53,6 +53,34 @@ describe("BananService") {
 
 Each `it` block should have only one assertion. This is in line with the _Isolated_ principle.
 
+```swift
+describe("Movie") {
+  context("when initializing from dictionary") {
+  
+    let name = "The Dark Knight Rises"
+    let genre = "Action"
+    let year = 2012
+    let movie: Movie!
+    
+    beforeEach {
+      movie = Movie(dict: ["name": name,
+                           "genre": genre,
+                           "year": year])
+    }
+
+    it("maps name property") {
+      expect(movie.name) == name
+    }
+    it("maps genre property") {
+      expect(movie.genre) == genre
+    }
+    it("maps year property") {
+      epxect(movie.year) == year
+    }
```

Comment:
👍 👍 👍 

I like how there's no newline between the `it` blocks, I think it reinforces the fact that they are together.

This opens the question of how is our guideline for new lines within tests thought... 🤔 

Maybe for consistency and simplicity "one new line between blocks" should be applied as the default?

_I started writing an example but stopped as I think this deserves a dedicated issue_.